overlord/snapstate/snapstate.go

```diff
@@ -302,8 +306,68 @@ func Remove(s *state.State, name string) (*state.TaskSet, error) {
 
 // Rollback returns a set of tasks for rolling back a snap.
 // Note that the state must be locked by the caller.
-func Rollback(s *state.State, snap, ver string) (*state.TaskSet, error) {
-	return nil, fmt.Errorf("rollback not implemented")
+func Rollback(s *state.State, name, ver string) (*state.TaskSet, error) {
```

Comment:
I don't have a strong opinion either way, but if keep it to just one rollback recorded, we should error here if a 2nd is tried. Also do we need to reset RollbackR somewhere?